pkg/virt-controller/watch/application.go

```diff
@@ -961,6 +958,12 @@ func (vca *VirtControllerApp) AddFlags() {
 
 	flag.IntVar(&vca.cloneControllerThreads, "clone-controller-threads", defaultControllerThreads,
 		"Number of goroutines to run for clone controller")
+
+	flag.StringVar(&vca.productName, "product-name", "",
```

Comment:
Where do you pass `vca.productName` to the templator?

Reply:
this https://github.com/kubevirt/kubevirt/pull/10204/files#diff-603814ab7b1e6c7a99ebf35091dcb751056ad1d861bc45d424c89d8b799c6e1dR609-R610, you mean?

Reply:
Seems to me we have a bug in KV propagation to the job. Nevertheless, we are trying to use Kubernetes labels which I think are more suitable for the user use case.